openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,30 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+
+describe('LatestUpdate tests', () => {
+  function createLatestUpdate() {
+    new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+  }
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      $.cookie('update-message', '');
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+    });
+
+    it('Test dismiss button', () => {
+      createLatestUpdate();
+      expect($('.update-message').attr('style')).toBe(undefined);
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+    });
+
+    it('Test cookie hides update', () => {
+      $.cookie('update-message', 'hide');
```

Comment:
I think the question is about the intent of testing the cookie.  In english, the things we might want to test:
1. That the cookie is set to expire in 1 day, and
2. The after the cookie expires, that the message will start to appear again.

For the first, you could add to the existing test an assertion that there is no cookie before, and that there is a cookie after, and that its expiration is ~a day from the dismiss time.

For the second, you could have the current cookie test exactly as you first coded it, but then delete the cookie and create a new message and show that it starts to appear.

I imagine the second would be simple to implement.  The first would possibly be simple to assert that the cookie exists or doesn't exist.  The date based check might be more complicated, and I'll leave that to you.

Does this make sense?